go.mod

```diff
@@ -37,9 +37,9 @@ require (
 )
 
 replace (
-	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
-	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
-	github.com/docker/docker => github.com/moby/moby v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible //nolint
+	//github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
```

Comment:
@tanvigour are you looking into the build failures? It looks like commenting this out might be creating a missing dependency.

Reply:
Yes, I am looking into it. So earlier it seemed liked the replacement wasn't really being used in the cli codebase and since go linter new version doesn't like replacements thought commenting it might be harmless. But guess there might be still some references to it somewhere that I missed.